cohort-evaluator/src/main/java/com/ibm/cohort/cql/translation/CqlToElmTranslator.java

```diff
@@ -93,6 +88,34 @@ public CqlTranslationResult translate(CqlLibrary primaryLibrary, CqlLibrarySourc
 
         return new CqlTranslationResult(translatedLibrary, dependencies);
     }
+
+    public CqlTranslatorOptions getOptions() {
+        return CqlTranslatorOptions.defaultOptions()
+                 .withOptions(CqlTranslator.Options.EnableResultTypes)
+                 .withAnalyzeDataRequirements(true)
+                 //.withCollapseDataRequirements(true)
```

Comment:
Pedantic: Might be worth adding a comment detailing why this bit is commented out.

Reply:
I've left these in place with both uncommented. I believe they are no longer relevant because we are no longer using the elm-fhir project and that is the only thing that references them, but I need to do some investigation to confirm that. I'm going to poke Bryn and see if he can clarify.